www-src/app/replicator/main.coffee

```diff
@@ -638,7 +638,9 @@ module.exports = class Replicator extends Backbone.Model
 
         ReplicationLauncher = require "./replication_launcher"
         @replicationLauncher = new ReplicationLauncher @config, app.router
-        @replicationLauncher.start options, callback
+        @replicationLauncher.start options, =>
+            @stopRealtime()
```

Comment:
Ok, that's not obvious why you are calling stopRealtime here.
It's just to clean up the @replicationLauncher objects ?